### DISCUSSION TITLE
Better RNG

### DISCUSSION THREAD

**Discussion by mrspudcake** [godotengine/godot-proposals#9331] [Scripting]
The RNG feature is unusually complex. Any simplifications would be helpful. Something like "randi(5, 10)" picking between a random integer between 5 and 10.

**Comment by mrspudcake** [comment]
If I'm wrong, tell me

**Comment by AThousandShips** [comment]
What's wrong with [`randi_range`](https://docs.godotengine.org/en/stable/classes/class_%40globalscope.html#class-globalscope-method-randi-range)? If it's the fact that you can't do `randi(5, 10)` that's just a limitation of how GDScript works, you can't have overloads (also it's *much* more clear this way, and easier to use, so using a range with that would be *worse* than it is now)
